### DISCUSSION TITLE
Airflow inbuilt public.dag tables last_expired field

### DISCUSSION THREAD

**Discussion by akshaymetange914** [apache/airflow#22639] [Q&A]
From airflow ui , i am getting my all dags displayed i need to work on scenario where all the disabled dag i can capture and send all of them on email as alert .i had developed that functionality but when i try to manually disable the dag by turning dag to off its giving the result for dag as "None" instead i should get the last modified date for the recently disabled dag . Why i am getting the disable dag status as "None" i am not getting can comunity as well q & a team would give me reason why its showing disabled dag last_modified time as "None"? here i am attaching snapshot for my recent disable dag which has last_modified status as "None".

![Screenshot from 2022-03-31 09-55-36](https://user-images.githubusercontent.com/85549598/160976524-21578e53-030c-4b31-90b4-91c41a1385a1.png)
![Screenshot from 2022-03-31 09-55-36](https://user-images.githubusercontent.com/85549598/160976543-d4d5a05d-7a7d-48ec-ae1f-49c3861a45bb.png)

**Comment by potiuk** [comment]
I believe those were not part of the API https://github.com/apache/airflow/pull/22637 - seems it will be aded for 2.3.0

**Reply by akshaymetange914** [reply]
actually i am developing the functionality that all the disable dag i should capture on basis is_paused field .if is_paused value will be true then it will consider it as disable dag but in addition to that i do need the disable dag which got disabled in last 2hr period of time  for tha i am using last_expired field .So for testing purpose i had made one of dag manually disabled but it seems that last_expired field should get updated with time/date but its changing the field status to None .Hope you got the point what i am explaining can you  guide why it is not giving time/date when i disable dag manually through airflow ui.

**Reply by potiuk** [reply]
I believe those were not part of the API https://github.com/apache/airflow/pull/22637 - seems it will be aded for 2.3.0
